test/e2e/node_feature_discovery.go

```diff
@@ -393,7 +393,7 @@ func cleanupNode(cs clientset.Interface) {
 
 			// Remove annotations
 			for key := range node.Annotations {
-				if strings.HasPrefix(key, master.AnnotationNs) {
+				if strings.HasPrefix(key, master.AnnotationNsBase) {
```

Comment:
in case instance is used it will break cleanup right ?

if/when support is added to instance flag in e2e test this would need to be modified.

Reply:
Yes, correct. My plan was to update/extend the e2e tests in a separate PR